docs/how_to_guides/record.md

```diff
@@ -446,3 +446,25 @@ updated_data = [
 ]
 dataset.records.update(records=updated_data)
 ```
+!!! note "Update the metadata"
+    To update the metadata of a record, you can iterate over the records and update the metadata dictionary by key, attribute or using `metadata.update`. After that, you should update the records in the dataset.
+
+    ```python
+    updated_records = []
+    for record in dataset.records():
+
+        # By key
+        record.metadata["my_metadata"] = "new_value"
+        record.metadata["my_new_metadata-key"] = "new_value"
+
+        # By attribute
+        record.metadata.my_metadata = "new_value"
+        record.metadata.my_new_metadata = "new_value"
```

Comment:
I would skip this way, even if it's supported. Just to show the metatada to users as a dictionary.
```suggestion
```